### ISSUE TITLE
Allow ALT text to be turned off

### ISSUE THREAD

**Issue by cdowdy** [cdowdy/boltresponsiveimages#15]
In some instances alt text isn't needed or the filename (which is currently used if no alt text is supplied) isn't sufficient or its redundant

http://webaim.org/techniques/alttext/#context
